wallet/src/actors/app/methods.rs

```diff
@@ -749,6 +756,58 @@ impl App {
             .do_send(jsonrpc::Subscribe(request, recipient));
     }
 
+    /// Send syncStatus request to the node every 10 seconds and send
+    /// NodeDisconnected event if error in the response
+    pub fn periodic_node_request(&self, ctx: &mut <Self as Actor>::Context) {
+        let method = "syncStatus".to_string();
+
+        let wallets: Vec<types::SessionWallet> = self
+            .state
+            .wallets
+            .iter()
+            .map(|(_, wallet)| wallet.clone())
+            .collect();
+
+        let events = Some(vec![types::Event::NodeDisconnected]);
+        let req = types::RpcRequest::method(method)
+            .timeout(self.params.requests_timeout)
+            .params(())
+            .expect("params failed serialization");
+
+        log::debug!("Sending periodic request: {:?}", req);
+
+        let f = self
+            .get_client()
+            .actor
+            .send(req)
+            .flatten()
+            .inspect(|res| {
+                log::debug!("Periodic request result: {:?}", res);
+                let status = serde_json::from_value::<types::SyncStatus>(res.clone());
+                log::debug!("The result of the node status is {:?}", status.unwrap());
+            })
+            .into_actor(self)
+            .map_err(move |err, act, _ctx| {
+                log::warn!("Periodic request failed: {}", &err);
+                log::info!("The node in disconnected");
+                // Notify that the node is disconnected
+                for wallet in &wallets {
+                    let sink = act.state.get_sink(&wallet.session_id);
+                    act.params
+                        .worker
+                        .do_send(NotifyStatus(wallet.clone(), sink, events.clone()))
+                }
+            })
+            .map(|_res, _act, _ctx| {
+                // TODO: send event with node state
+            });
```

Comment:
```suggestion
```

Reply:
This will be used later to send a notification to Sheikah